walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -274,7 +274,7 @@ public void notEnoughRoom() {
   }
 
   @Test
-  public void everyAttendeeIsConsideredInclOptionalAttendeeWithNoAvailability() {
+  public void allAttendeesConsideredReturnsTimesIgnoringOptionalAttendee() {
```

Comment:
I really like these names.
Just one suggestion for each of them since they all return the time:
```suggestion
  public void allAttendeesConsideredIgnoresOptionalAttendee() {
```

Reply:
Also, if there's any conditions specific to when the optional attendees get ignored, you'd want that in too.